index.html

```diff
@@ -253,6 +253,21 @@ <h2 class="media_header">LET'S CONNECT</h2>
         </div>
     </section>
     </div>
+    <footer>
+			<form action="https://formspree.io/f/xwkyyyvp" method="post">
+				<h2>Contact me</h2>
+				<p class="contact-infor">
+					If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+				</p>
+				<input name="name" type="text" placeholder="Name">
+				<input name="email" type="email" placeholder="Email">
+				<textarea name="message" id="message" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] Kindly be noticed that the message should have maxlength="500" and required attributes.